stage1/init/common/pod.go

```diff
@@ -938,18 +923,76 @@ func GetMachineID(p *stage1commontypes.Pod) string {
 	return "rkt-" + p.UUID.String()
 }
 
-// GetAppCapabilities is a filter which returns a string slice of valid Linux capabilities
+// getAppCapabilities is a filter which returns a string slice of valid Linux capabilities
```

Comment:
This function does a lot of things it didn't do before, maybe you can document it a bit better?